compiler/luci/import/src/Importer.cpp

```diff
@@ -268,6 +268,10 @@ Importer::Importer()
 
 std::unique_ptr<Module> Importer::importModule(const circle::Model *model) const
 {
+  assert(model);
```

Comment:
This is checked in `importModule(const uint8_t *data, size_t size)`

Reply:
Thanks for pointing this :)